array-map/main.js

```diff
@@ -0,0 +1,19 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const doubled = numbers.map(x => x * 2);
+console.log(doubled);
+const prices = numbers.map(x => '$' + x.toPrecision() + '.00');
```

Comment:
This works for this exercise but in general when you want to have a specific amount of decimal places on a number the `toFixed` method is a good way to go.

```JAVASCRIPT
x => '$' + x.toFixed(2)
```